bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java

```diff
@@ -267,6 +282,31 @@ else if (String.class.isAssignableFrom(type))
         }
     }
 
+    private String promptForListItem(String message, Collection<String> items, String defaultValue)
```

Comment:
Otherwise this is fine. I guess this is the only command that prompts at the moment.